temba/orgs/context_processors.py

```diff
@@ -45,7 +45,7 @@ def __contains__(self, perm_name):
 def user_orgs_for_brand(request):
     if hasattr(request, "user"):
         if not request.user.is_anonymous:
-            user_orgs = request.user.get_user_orgs(request.branding.get("brand"))
+            user_orgs = request.user.get_user_orgs(request.branding.get("brand")).filter(is_active=True)
```

Comment:
@srugano - You should filter if the organization is suspended or not, this `is_active=True` filter already exists inside the function `get_user_orgs`